src/groups/validation_group.jl

```diff
@@ -0,0 +1,760 @@
+"""
+    ValidationLieGroup{L<:AbstractLieGroup} <: AbstractLieGroup
+
+A Lie group to add tests to input parameters and output values of functions defined
+for [`LieGroups`](@ref).
+
+
+Using the `ignore_contexts` keyword allows to specify a single `Symbol` or a vector of `Symbols`
+Of which contexts to ignore.
+
+Current contexts are
+* `:All`: disable all checks
+* `:Point`: checks for points
+* `:Algebra`: checks related to the [`LieAlgebra`](@ref)
+* `:Output`: checks for output
+* `:Input`: checks for input variables
+
+# Fields
+
+* `lie_group::L` the [`AbstractLieGroup`](@ref) to be decorated
+* ``
```

Comment:
Other fields seem to be missing, though we could probably just link to `ValidationManifold` for their description`?

Reply:
Ah I forgot to add them, yes it is basically the same fields (again) as for the Validation one, and I would even try to keep both parallel – since that way one can pass down the same things and validate on M as well – or not – if one wants to.

Reply:
Fixed it :)